src/SearchButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function SearchButton() {
+  return (
+    <div className="search-row">
+      <input
+        type="text"
+        id="customerName"
+        className="form-control"
+        placeholder="Customer name"
+      />
+      <button className="btn btn-success">Search</button>
```

Comment:
You have the right idea with this but the question was asking you to extract the `<button>` element only, so you only needed this line in your `return` statement